docs/smartfix_coding_agent.md

```diff
@@ -0,0 +1,333 @@
+          base_branch: '${{ github.event.repository.default_branch }}' # This will default to your repo default branch (other common base branches are 'main', 'master' or 'develop')
+
+          # Required Runtime Configuration
+          build_command: 'mvn clean install' # Or the build command appropriate for your project.  SmartFix will use this command to ensure that its changes work correctly with your project.
+
+          # LLM Configuration (Bring Your Own LLM)
+          # Choose ONE LLM provider and configure its credentials
+          # Recommended: Anthropic Claude Sonnet
+
+          # Claude Via Direct Anthropic API
+          # agent_model: 'anthropic/claude-3-7-sonnet-20250219' # Check LiteLLM docs for exact model string
+          # anthropic_api_key: ${{ secrets.ANTHROPIC_API_KEY }}
+
+          # Claude Via AWS Bedrock
+          # Setup AWS credentials in the earlier "Configure AWS Credentials" step.
+          agent_model: 'bedrock/us.anthropic.claude-3-7-sonnet-20250219-v1:0' # Example for Claude Sonnet on Bedrock
+
+          # Experimental: Google Gemini Pro
+          # agent_model: 'gemini/gemini-2.5-pro-preview-05-06' # Check LiteLLM docs for exact model string
+          # gemini_api_key: ${{ secrets.GEMINI_API_KEY }}
+
+          # Other Optional Inputs (see action.yml for defaults and more options)
+          # formatting_command: 'mvn spotless:apply' # Or the command appropriate for your project to correct the formatting of SmartFix's changes.  This ensures that SmartFix follows your coding standards.
+          # max_open_prs: 5 # This is the maximum limit for the number of PRs that SmartFix will have open at single time
+
+  handle_pr_merge:
+    name: Handle PR Merge
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.merged == true && contains(join(github.event.pull_request.labels.*.name), 'contrast-vuln-id:VULN-')
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Notify Contrast on PR Merge
+        uses: Contrast-Security-OSS/contrast-ai-smartfix-action@v1 # Replace with the latest version
+        with:
+          run_task: merge
+          # --- GitHub Token ---
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          # --- Contrast API Credentials ---
+          contrast_host: ${{ vars.CONTRAST_HOST }}
+          contrast_org_id: ${{ vars.CONTRAST_ORG_ID }}
+          contrast_app_id: ${{ vars.CONTRAST_APP_ID }}
+          contrast_authorization_key: ${{ secrets.CONTRAST_AUTHORIZATION_KEY }}
+          contrast_api_key: ${{ secrets.CONTRAST_API_KEY }}
+        env:
+          GITHUB_EVENT_PATH: ${{ github.event_path }}
+
+  handle_pr_closed:
+    name: Handle PR Close
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.merged == false && contains(join(github.event.pull_request.labels.*.name), 'contrast-vuln-id:VULN-')
```

Comment:
Does this need to be updated to use:
`github.event.pull_request.merged == true && (contains(github.event.pull_request.head.ref, 'smartfix/remediation-')`